### ISSUE TITLE
Using additional keys in config not allowed?

### ISSUE THREAD

**Issue by FallingSnow** [Antergos/lightdm-webkit2-greeter#93]
Can we add our own keys to the `/etc/lightdm/lightdm-webkit2-greeter.conf`? Currently it seems to throw an error.

**Comment by lots0logs** [comment]
No, currently that isnt possible. I think we can certainly add support for custom keys. Probably do it as a separate section named `theme`.

**Comment by FallingSnow** [comment]
I agree, I just figured it would already work and I was just doing it wrong.

**Comment by NoiSek** [comment]
I've been meaning to ask you about that @lots0logs, is there any guarantee for themes that localStorage is 'safe'?

In what situations is that cache wiped? Is it preserved across lightdm-webkit upgrades?

I'm planning on putting a lot of user configurable data into localStorage in the near future, so I'm trying to figure out whether or not it makes sense to serialize it into a separate backup config somewhere.

**Comment by lots0logs** [comment]
The same rules apply in this case as with the web. Local storage is never guaranteed and can be cleared by the user at any time. Though in reality, most users are not going to go out of their way to clear the local storage of the greeter. Even if they do, it simply means their settings will go back to default. The theme should not break just because the storage was cleared.

**Comment by FallingSnow** [comment]
Are custom keys supported yet?
